webapp/src/main/webapp/vue-apps/activity-stream/components/toolbar/ActivityComposerDrawer.vue

```diff
@@ -345,6 +345,10 @@ export default {
       });
     },
     postMessage() {
+      if (this.loading) {
+        return;
+      }
+      this.loading = true;
```

Comment:
There is some cases where `loading` will remain true after the method finishes its execution (See `if` condition in line 380)

Reply:
yes true, thanks
Fixed now!